code/parse/sexp.cpp

```diff
@@ -13989,6 +13991,42 @@ void sexp_tech_add_intel(int node, bool xstr)
 	}
 }
 
+// wookieejedi, base copied from sexp_tech_add_intel()
+void sexp_tech_remove_intel(int node, bool xstr)
```

Comment:
Considering that the add and remove operations are almost entirely identical, except for the flags and messages, I would ask that the two functions be combined together and a `bool add` parameter be inserted before the `bool xstr` parameter.  (I'm in the process of combining a bunch of other programs as part of my sexp caching overhaul, but that isn't PR-ready yet.)  Take a look at the dock/undock sexps to see how I handled warning messages for the different sexps.